app/hotels/src/singleHotel/SingleHotelContainer.js

```diff
@@ -13,6 +13,7 @@ import HotelsContext, {
 import SingleHotelContext, {
   type State as SingleHotelState,
 } from '../navigation/singleHotel/SingleHotelContext';
+import Stay22SingleHotel from './Stay22SingleHotel';
```

Comment:
We could even lazy load it using `Suspense` and `lazy`; what do you think? Does it make sense in RN?

Reply:
> Does it make sense in RN?

No 😄